src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,454 @@
+    surface_type: int
+        the type of roof we are calculating the metal load for. Some coefficients depend on this.
+
+    Returns
+    -------
+    (float, float)
+       Returns the total copper and zinc loads from the given parameters (micrograms)
+    """
+    # Define constants in a list
+    b = []
+    c = []
+
+    match surface_type:
+        case 1:
+            b = [2, -2.8, 0.5, 0.217, 3.57, -0.09, 7, -3.73]
+            c = [50, 2600, 0.1, 0.01, 1, -3.1, -0.007, 0.056]
+        case 2:
+            b = [100, -2.8, 1.372, 0.217, 3.57, -1, 275, -3.3]
+            c = [-0.1, 2, 0.1, 0.01, 0.8, -1.3, -0.007, 0.056]
+        case 3:
+            b = [2, -2.8, 0.5, 0.217, 3.57, -0.09, 7, -3.73]
+            c = [910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.990]
+        case _:
+            log.error(
+                f"Given surface type is not valid for computing total metal load. Needed a roof, but got {SurfaceType(surface_type).name}.")
+    # Define the initial and second stage metal concentrations (X_0 and X_est)
+    initial_copper_concentration = (b[0] * rainfall_ph ** b[1]) * (b[2] * antecedent_dry_days ** b[3]) * (
+            b[4] * average_rain_intensity ** b[5])
+    second_stage_copper = b[6] * rainfall_ph ** b[7]
+
+    initial_zinc_concentration = (c[0] * rainfall_ph + c[1]) * (c[2] * antecedent_dry_days ** c[3]) * (
+            c[4] * average_rain_intensity ** c[5])
+    second_stage_zinc = c[6] * rainfall_ph + c[7]
+
+    # Define Z as per experimental data
+    z = 0.75
+
+    # Define K, the wash off coefficient. TODO: find out what the k parameter should be
+    k = 1
+
+    # Initialise total copper and zinc loads as a guaranteed common factor
+    total_copper_load = initial_copper_concentration * surface_area * (1 / k)
+    total_zinc_load = initial_zinc_concentration * surface_area * (1 / k)
+
+    # Calculate total metal loads, where the method depends on if Z is less than event_duration
+    if event_duration <= z:
+        factor = (1 - math.exp(k * average_rain_intensity * event_duration))
+        total_zinc_load *= factor
+        total_copper_load *= factor
+    else:
+        factor = (1 - math.exp(k * average_rain_intensity * z))
+        bias_factor = average_rain_intensity * (event_duration - z)
+        total_zinc_load = total_zinc_load * factor + second_stage_zinc * surface_area * bias_factor
+        total_copper_load *= total_copper_load * factor + second_stage_copper * surface_area * bias_factor
+
+    return total_copper_load, total_zinc_load
+
+
+def total_metal_load_road_carpark(tss_surface):
+    """
+    Calculate the total metal load for car parks and roads from their total suspended solids.
+
+    Parameters
+    ----------
+    tss_surface: float
+        total suspended solids of this surface
+
+    Returns
+    -------
+    float, float
+       Returns the total copper and zinc loads for this surface
+    """
+    # Define constants
+    d = 0.441
+    e = 1.96

```

Comment:
better to avoid single letter variable names, if you know what it represents then you can write something more explicit

Reply:
d and e are "dimensionless proportionality coefficients derived from experimental data". I am not sure what else to call them, maybe "proportionality_constant"?